#!/usr/bin/env python
PACKAGE = "bobcat_model"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Wheel_conntrol_P",	double_t,	0,	"Porptional",	25000,	0,  50000)
gen.add("Wheel_conntrol_I",	double_t,	0,	"Porptional",	0.0,    0,  100000)
gen.add("Wheel_conntrol_D",	double_t,	0,	"Porptional",	0.0,	0,  100000)	

gen.add("Power",	double_t,	0,	"Power multiplier",	160,	0,  300)	
gen.add("Damping",	double_t,	0,	"Velocity Damping",	160,	0,  300)	

gen.add("Command_Linear_Noise",	   double_t,	0,	"Porptional",	0.0,	0,  1)
gen.add("Command_Angular_Noise",   double_t,	0,	"Porptional",	0.0,	0,  1)

exit(gen.generate(PACKAGE, "bobcat_model_reconfig_node", "bobcat_model"))
